extensions/wikia/MonetizationModule/MonetizationModuleHelper.class.php

```diff
@@ -347,15 +334,13 @@ public static function insertIncontentUnit( $body, &$monetizationUnits ) {
 			if ( $posTOC === false ) {
 				// TOC not exist. Insert the ad above the 2nd <H2> tag.
 				$body = substr_replace( $body, $monetizationUnits[self::SLOT_TYPE_IN_CONTENT], $pos2, 0 );
-				wfProfileOut( __METHOD__ );
 				return $body;
 			} else {
 				// TOC exists. Check for the 3rd <H2> tag.
 				$pos3 = strpos( $body, self::IN_CONTENT_KEYWORD, $pos2 + $keywordLength );
 				if ( $pos3 !== false ) {
 					// The 3rd <H2> tag exists. Insert the ad above the 3rd <H2> tag.
 					$body = substr_replace( $body, $monetizationUnits[self::SLOT_TYPE_IN_CONTENT], $pos3, 0 );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$body". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)